### DISCUSSION TITLE
Get unique values filter

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#1868] [Q&A]
How to get unique rows from pocketbase in a filter?

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean. Could you clarify a little more on your requirement?

If you are asking for aggregations, then No, at the moment this is not possible via the web APIs, but it will be available in the upcoming v0.13.0 release (_most likely next week or the week following it_).

**Comment by starredev** [comment]
I mean, in my collection migrations I have multiple rows with the same value in one column. I want to filter out the duplicates. In MySQL you could do distinct

**Reply by ganigeorgiev** [reply]
I'm sorry, I still don't the understand what you are trying to do.

Could you elaborate a little more and provide a code sample (pseudo-code is also ok)?

Is this for a migration script? If yes, you can specify `DISTINCT` on individual column by executing a raw db query - https://pocketbase.io/docs/database/#query-builder.

If you want to apply a custom query via the web APIs, then as mentioned in the previous post, this at the moment is not supported but will be available with v0.13.0 and the new "View" collection type.
